### DISCUSSION TITLE
Submenu for Lyre

### DISCUSSION THREAD

**Discussion by Zavarox** [runelite/runelite#17908] [Ideas]
I would like a plugin or update for the menu entry swapper to have submenus for teleports on the Lyre, for easy left click or shift click to places such as neitiznot. Is that possible within the game and/or rules?

**Comment by YvesW** [comment]
Jagex only allows this for capes, not for the lyre.

**Reply by Zavarox** [reply]
Awh man thats unfortunate, thanks for the reply.
